constellation-app/constellation
# Update to wrap text

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Added code in TableCellFactory to wrap text within table column width.
Added appropriate selectors to set new Text node text colours to behave as before.


### Alternate Designs
Setting text wrap in css expanded the column width to the length of the text, which is not preferred behaviour.
This solution allowed text wrap within the table column width setting.


### Why Should This Be In Core?



### Benefits
To be able to view the text to be viewed easily (especially if it has no whitespace)


### Possible Drawbacks



### Verification Process

1. Open Constellation and open a graph
2. Create a node, make the identifier a really long name
3. Open Table View, switch to node view.
Observe that the identifier column holds the entire name wrapped within the column.



### Applicable Issues
#1983
